### PR TITLE
Update for Qt (5.14) and qutebrowser (1.13.0).

### PR DESCRIPTION
**Pull request by ysiraichi**
Not sure if this is the way to do it, but... I have been wanting to use updated versions of _qutebrowser_ via flatpak, so I made some changes to this base file. Anyway, since I already have it, I figured I would just share here.

### CODE REVIEW

File: launch.sh [hunk 1]
```diff
@@ -0,0 +1,3 @@
+#! /bin/sh
+export QTWEBENGINEPROCESS_PATH=/app/bin/QtWebEngineProcess
```
Comment (tinywrkb) [inline launch.sh:2]: Why aren't you adding instead `--env=QTWEBENGINEPROCESS_PATH=/app/bin/QtWebEngineProcess` to the `finish-args` in the manifest?

Comment (ysiraichi) [reply launch.sh]: No particular reason. I just didn't know about it (sorry about that).
If you have any other observations, please do.
This is my first time messing with flatpak, so there may be lots of things to improve.

**Comment by tinywrkb** [comment]
@ysiraichi if you look [here](https://github.com/flathub/io.qt.qtwebengine.BaseApp/blob/9748b557c6ff4eba2d9651a4fba1c669ff040416/io.qt.qtwebengine.BaseApp.json#L123) you'll notice `/app/cleanup-BaseApp.sh`. You should make use of it with something like
```
cleanup-commands:
  - /app/cleanup-BaseApp.sh
  - rm -f /app/cleanup-BaseApp.sh 
```

**Comment by ysiraichi** [comment]
@tinywrkb thanks for the tip.

**Comment by tinywrkb** [comment]
No problem, I actually tried bumping the runtime but failed because I didn't set `QMAKE_INCDIR` properly.

**Comment by ysiraichi** [comment]
To be fare, I also got into the same problem, which I could solve because [some guy had the same error, and shared how he solved it.](https://github.com/flathub/io.qt.qtwebengine.BaseApp/issues/2)

**Comment by ysiraichi** [comment]
So... I don't really know how this works, but can we merge it?

**Comment by tinywrkb** [comment]
@ysiraichi 
It seems like the maintainer is neglecting this package. Open a bug report [here](https://github.com/flathub/flathub/issues) and if you use this app regularly then maybe ask to co-maintain the package.  
The perfect solution would be for the app developers to maintain the package but they didn't step up yet.

**Comment by bilelmoussaoui** [comment]
bot, build

File: org.qutebrowser.qutebrowser.yml [hunk 2]
```diff
@@ -127,16 +139,14 @@ modules:
...
 
   - name: qutebrowser
     buildsystem: simple
     build-commands:
       - make --file misc/Makefile install PREFIX=/app
     sources:
       - type: archive
-        url: https://github.com/qutebrowser/qutebrowser/releases/download/v1.7.0/qutebrowser-1.7.0.tar.gz
-        sha256: f1f01b33669a9b08071997bb2642d46a1e5be9e293bdeb817d83db8dc5aad273
-      - type: patch
```
Comment (bilelmoussaoui) [inline org.qutebrowser.qutebrowser.yml:141]: Any reason for removing just the patch from here?

Comment (ysiraichi) [reply org.qutebrowser.qutebrowser.yml:141]: Two reasons:
- the patch failed to be applied to v1.13.0; and
- at the time, I got to the conclusion that it didn't make any difference (don't ask me why).

Now, I have created a new patch for that, and included it in the yml.

**Comment by tinywrkb** [comment]
Closed with #10 which is based on the changes here.

@ysiraichi thank you for your efforts. FYI I added you as a co-author of #10.

**Pull request closed without merging**